docs/excel/custom-functions-errors.md

```diff
@@ -33,15 +30,17 @@ function getCity(zipCode: string): string {
 
 ## The CustomFunctions.Error object
 
-The `CustomFunctions.Error` object is used to return an error back to the cell. When you create the object, specify which error you want to use by using one of the following `ErrorCode` enum values.
+The `CustomFunctions.Error` object is used to return an error back to the cell. When you create the object, specify which error you want to use by choosing one of the following `ErrorCode` enum values.
```

Comment:
What do you think about linking directly to the ref docs for this Error object? I always like doing that for the first reference to an object in the article.
```suggestion
The [CustomFunctions.Error](/javascript/api/custom-functions-runtime/customfunctions.error) object is used to return an error back to the cell. When you create the object, specify which error you want to use by choosing one of the following `ErrorCode` enum values.
```

Reply:
Great idea!